Cisco/cisco-meraki/_meta/manifest.yml

```
@@ -0,0 +1,9 @@
+uuid: 995d7daf-4e4a-42ec-b90d-9af2f7be7019
+name: Cisco Meraki
+slug: cisco-meraki
+
+description: Meraki MX Security Appliance
```

As this format address Meraki MX, I suggest the following changes:
```suggestion
name: Cisco Meraki MX
slug: cisco-meraki-mx

description: Cisco Meraki MX is multifunctional security and SD-WAN enterprise appliance with a wide set of capabilities.
```